presidio-analyzer/conf/analyzer.yaml

```diff
@@ -0,0 +1,93 @@
+recognizer_registry:
+  recognizers: 
+    - name: CreditCardRecognizer
+      supported_languages: 
```

Comment:
Yeah, I like this approach. I kinda ignored the regex flag because I feel it'd be best if we parse it though from a string because int doesn't really express anything. But it's better than nothing.

Reply:
Yes I was wondering too about this one